knpu/start-project.md

```diff
@@ -25,34 +25,65 @@ doing *amazing* things with the Symfony framework.
 Let's get our first Symfony project rolling. Head over to Symfony.com and click
```

Comment:
:+1: I think that's a good idea :) - I'm adding it now. If you are just a reader, then it will save you from copying the URL - you can just click it